docs/reference/modules/indices/indexing_buffer.asciidoc

```diff
@@ -23,10 +23,3 @@ in the cluster:
 
     If the `index_buffer_size` is specified as a percentage, then this
     setting can be used to specify an absolute maximum.  Defaults to unbounded.
-
-`indices.memory.min_shard_index_buffer_size`::
```

Comment:
can we add this to the migrate doc that it has been removed without a replacement